### ISSUE TITLE
file manager: issues with deletion and restore

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2700]
Not sure if related to #2699 

In the file manager, if I try to delete a backup file I get this: 
<img width="1512" alt="Screenshot 2024-05-07 at 1 18 20 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/9eb90cc2-c375-4d3f-891b-c9de4cf0de4c">

But then the file shows up with the trash can next to it after the little message popup disappears. If I then try to restore it, I can this message and it does not in fact appear to have been restored: 

<img width="1512" alt="Screenshot 2024-05-07 at 1 19 45 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/cbb62f03-e4b9-4da2-9f9b-208247da4d3e">

**Comment by urapadmin** [comment]
ja, the delete does indeed not delete anything (can still be downloaded) and so all that happens afterwards is bogus

**Comment by urapadmin** [comment]
the issue itself is a lack of privileges. No right to delete a file. It would have been nice to get a better error message on that.

**Comment by urapadmin** [comment]
### Kiosk 1.6.26
- proper error message should reveal the issue now (the underlying problem is not a Kiosk problem but a matter of web / OS server config)
- the restore wasn't entirely wrong in this case. The file could have been restored. But then then the restore failed due to the same error and should have given a useful error message. That is now the case.
